pokebot.py

```diff
@@ -32,29 +33,56 @@ def on_exit() -> None:
 atexit.register(on_exit)
 
 
-def parse_arguments() -> argparse.Namespace:
+@dataclass
+class StartupSettings:
+    profile: 'Profile | None'
+    debug: bool
+    bot_mode: str
+    no_video: bool
+    no_audio: bool
+    emulation_speed: int
+    always_on_top: bool
+
+
+def parse_arguments() -> StartupSettings:
```

Comment:
There is now a huge traceback error + ungraceful exit if you launch with the `--help` or `-h` switch.